### ISSUE TITLE
Events feature request

### ISSUE THREAD

**Issue by supertopher** [cloudfoundry/bosh-notes#19]
For debugging it would be useful to show recent tasks filtered by deployment.  

Ie: Today i know that i have a problem that occurred with logging.  This problem is probably in the deployment `cf-cfapps-io2` However, at least as far as i know i cannot looks at recent tasks that have occurred specifically to this deployment.  

alternatively, if in the recent tasks we had another field shown for deployment we would be able to parse this on our own using AWK or other UNIX tools.  Screenie below to illustrate what i am talking about here 

![screen shot 2016-02-26 at 11 12 19 am](https://cloud.githubusercontent.com/assets/2591516/13362535/016e238e-dc7a-11e5-9439-e4ee82642ff2.png)

Thx for consideration =D

**Comment by allomov** [comment]
I guess this issue can be closed since bosh cli (since [1.3213](https://github.com/cloudfoundry/bosh/commit/ebedb0795c67adddfb7d55bf0317884da4502110) I guess) has such feature:
```
$ bosh help tasks recent
tasks recent [<count>] [--no-filter] [--deployment DEPLOYMENT_NAME]
    Show <number> recent tasks
    --no-filter                                               Include all task types (ssh, logs, vms, etc)
    --deployment DEPLOYMENT_NAME                              Show tasks for given deployment

tasks [--no-filter] [--deployment DEPLOYMENT_NAME]
    Show running tasks
    --no-filter                                               Include all task types (ssh, logs, vms, etc)
    --deployment DEPLOYMENT_NAME                              Show tasks for given deployment
```

What do you think @cppforlife ?

**Comment by allomov** [comment]
I guess this issue can be closed since bosh cli (starting from [1.3213](https://github.com/cloudfoundry/bosh/commit/ebedb0795c67adddfb7d55bf0317884da4502110)) has such feature:
```
$ bosh help tasks recent
tasks recent [<count>] [--no-filter] [--deployment DEPLOYMENT_NAME]
    Show <number> recent tasks
    --no-filter                                               Include all task types (ssh, logs, vms, etc)
    --deployment DEPLOYMENT_NAME                              Show tasks for given deployment

tasks [--no-filter] [--deployment DEPLOYMENT_NAME]
    Show running tasks
    --no-filter                                               Include all task types (ssh, logs, vms, etc)
    --deployment DEPLOYMENT_NAME                              Show tasks for given deployment
```
What do you think @cppforlife ?

**Comment by cppforlife** [comment]
@allomov ah that's true. seems like it's done.
